Tests/scripts/gitlab_slack_notifier.py

```diff
@@ -134,23 +167,22 @@ def bucket_upload_results(bucket_artifact_folder, should_include_private_packs:
     if successful_packs:
         steps_fields += [{
             'title': f'Successful {marketplace_name} Packs:',
-            'value': ', '.join(sorted([pack_name for pack_name in {*successful_packs}], key=lambda s: s.lower())),
+            'value': ', '.join(sorted({*successful_packs}, key=lambda s: s.lower())),
             'short': False
         }]
 
     if failed_packs:
         steps_fields += [{
             'title': f'Failed {marketplace_name} Packs:',
-            'value': ', '.join(sorted([pack_name for pack_name in {*failed_packs}], key=lambda s: s.lower())),
+            'value': ', '.join(sorted({*failed_packs}, key=lambda s: s.lower())),
             'short': False
         }]
 
     if successful_private_packs and should_include_private_packs:
         # No need to indicate the marketplace name as private packs only upload to xsoar marketplace.
```

Comment:
Is there a distinction between the XSOAR and XSOAR-NG/XSOAR-SAAS marketplaces?
If yes, then maybe we need to indicate the marketplace here?

Reply:
@tkatzir read the comment on line 149:

`        # No need to indicate the marketplace name as private packs only upload to xsoar marketplace.`